README.md

```diff
@@ -16,15 +16,73 @@
 # limitations under the License.
 #
 -->
+
 # Apache APISIX Dashboard
 
-Dashboard for [Apache APISIX](https://github.com/apache/incubator-apisix-dashboard)
+Dashboard for [Apache APISIX](https://github.com/apache/apisix-dashboard)
 
-## Deploy with Docker (currently)
+## Deploy with Docker
 
 Please refer to [Deploy with Docker README](./compose/README.md)
 
-## More
+## Deploy Manually
+
+### Build the manager-api
+
+The `manager-api` is used to provide APIs for Dashboard, just like a bridge between the Apache APISIX and the Dashboard. Here are the steps to build it manually:
+
+1. We need `MySQL/Golang` to be preinstalled.
+
+```sh
+# e.g Initialization for MySQL
+$ mysql –uroot –p123456 
+> source $PROJECT_ROOT_FOLDER/api/script/db/schema.sql
+```
+
+2. Start the Apache APISIX.
+
+[Please follow this guide](https://github.com/apache/apisix#configure-and-installation)
+
+3. Set environment variables
+
+According to your local deployment environment, modify the environment variables in `$PROJECT_ROOT_FOLDER/api/run/run.sh`
```

Comment:
what environment variables should user modify? I think it should be clearly written.

Reply:
the environment variables are about `mysql` and `APISIX`, which is depended by `manager-api`.
in `run.sh` we give the default config, maybe user need to modify
```sh
export MYSQL_SERVER_ADDRESS="127.0.0.1:3306"
export MYSQL_USER=root
export MYSQL_PASSWORD=123456
export SYSLOG_HOST=127.0.0.1
export APISIX_BASE_URL="http://127.0.0.1:9080/apisix/admin"
export APISIX_API_KEY="edd1c9f034335f136f87ad84b625c8f1"
```

Reply:
I change the `modify` to `check`, user can modify if necessary